wdl-lint/src/lib.rs

```diff
@@ -150,3 +152,32 @@ pub fn rules() -> Vec<Box<dyn Rule>> {
 
     rules
 }
+
+/// Gets the optional rule set.
+pub fn optional_rules() -> Vec<Box<dyn Rule>> {
```

Comment:
this is probably overkill since this will (hopefully) be a temporary home for shellcheck.